api/app/views/spree/api/promotions/handler.json.jbuilder

```diff
@@ -1,6 +1,6 @@
 # frozen_string_literal: true
 
 json.success(@handler.success)
-json.error(@handler.error)
```

Comment:
I think we should keep both `error` and `errors` here otherwise people consuming the API and relying on `error` in the JSON response will have their consumer broken.